client/consensus/babe/Cargo.toml

```diff
@@ -33,6 +35,9 @@ sp-runtime = { version = "2.0.0", path = "../../../primitives/runtime" }
 fork-tree = { version = "2.0.0", path = "../../../utils/fork-tree" }
 futures = "0.3.1"
 futures-timer = "3.0.1"
+jsonrpc-core = "14.0.3"
```

Comment:
Wouldn't it be cleaner to move to a separate sub-crate? This seems to grow significantly.

Reply:
should this go in `sc-rpc-api` or a sub-crate of `sc-consensus-babe`?

Reply:
Ping @tomusdrw

Reply:
Since it's a RPC extension I think it should be a sub-crate of `sc-consensus-babe`.